src/main/kotlin/com/nikitanazarov/coroutinestacks/Constants.kt

```diff
@@ -0,0 +1,11 @@
+package com.nikitanazarov.coroutinestacks
+
+class Constants {
```

Comment:
Let's remove this class and introduce corresponding constants closer to their use site. I will add more comments on that in this review.